source/changelog.txt

```diff
@@ -12,6 +12,42 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.3
+------
+
+*Released October 30, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1326` - Add ``shardedDataDistribution`` to
+  :method:`sh.status`.
+- Add individual evergreen test results with XUnit.
+- :issue:`MONGOSH-1730` - Ask for
+  ``tlsCertificateKeyFilePassword`` when it is needed.
+- :issue:`MONGOSH-1838` - Account for orphan documents in
+  :method:`~db.collection.getShardDistribution()` helper.
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1868` - Align database and collection
+  aggregate functions 
+- :issue:`MONGOSH-1608` - ``mongosh`` should throw an error when
+  trying to drop a non-primary index 
+- :issue:`MONGOSH-1868` - Add support for running aggregate
+  database with a single stage
+- Revert workaround for ``depcheck`` for ``searchParams`` type
+- Fix ``depcheck`` with a more permissive workaround for the ``searchParams`` type
```

Comment:
(same as above) this was a temporary internal fix that should be removed

Reply:
Fixed